src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
Looks really good to me, one last nitpick (please bear with me sorry :) ), but I think we should just name `QueueCallback` like `QueueWorker` (or just `QueueWork`?)  instead since it's not really specific to the callback anymore right?